app/globals.css

```
@@ -1299,4 +1488,114 @@ h6 {
+  background-image: none !important;
+  background-color: var(--color-brand-primary) !important;
+  border-top: none !important;
+  box-shadow: none !important;
+  /* Make the brand footer compact to visually align with the header */
+  padding-top: clamp(0.5rem, 1.5vw, 1rem) !important;
+  padding-bottom: clamp(0.5rem, 1.5vw, 1rem) !important;
+  /* Force fully opaque and remove decorative overlays behind the footer */
+  opacity: 1 !important;
+  background-blend-mode: normal !important;
+  /* Ensure the watermark or decorative background does not show through */
+  background-image: none !important;
```

Biome (2.1.2)

[error] Duplicate properties can lead to unexpected behavior and may override previous declarations unintentionally.

background-image is already defined here.

Remove or rename the duplicate property to ensure consistent styling.

(lint/suspicious/noDuplicateProperties)